src/utils/prepareConfigOverridesForKonductor.js

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
In other places we refer to Konductor as "Edge". Let's keep the same convention here: prepareConfigOverridesForEdge.js